test/component/component.compatibility.test.ts

```diff
@@ -42,9 +42,8 @@ for (let component of components) {
     });
 
     afterEach(async () => {
-      if (app) {
-         kill(app.pid);
-      }
+      if (app) app.kill('SIGINT');
```

Comment:
why not using kill(app.pid)? the kill lib is killing all the root and sub processes, which we need since we run the app via npm script